website/content/grants/contents.lr

```diff
@@ -1,49 +1,82 @@
 title: Förderprogramm
 ---
 body:
- 
-###Der Python Software Verband bietet ein Förderprogramm um die Python-Community im deutschsprachigen Raum in ihren vielfältigen Aktivitäten finanziell zu unterstützen.
 
+Der Python Software Verband bietet ein Förderprogramm um die Python-Community im deutschsprachigen Raum in ihren
```

Comment:
Wollen wir "im deutschsprachigen Raum" entfernen? Die Entfernung verpflichtet uns zu nichts, bietet aber mehr europäische Optionen.

Reply:
accordingly to the above: Our total funding program budget per year is approximately €15,000.